packages/icons/src/lib/svgs/rocket.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="24" height="23" viewBox="0 0 24 23" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0_17554_39523)">
+<g clipPath="url(#clip0_17554_39523)">
 <path d="M20.1903 0.45459H19.6303C14.7603 0.45459 10.4603 2.94459 7.9403 6.71459L5.3903 6.33459C4.8303 6.25459 4.2603 6.44459 3.8703 6.83459L1.4103 9.29459C0.950296 9.75459 0.780296 10.4046 0.950296 11.0246C1.1203 11.6446 1.6003 12.1246 2.2303 12.2846L5.6203 13.1546C5.6203 13.1546 5.6203 13.1946 5.6203 13.2246C5.5503 13.2646 5.4803 13.2946 5.4203 13.3546L2.9403 15.8346C2.6103 16.1646 2.6103 16.7046 2.9403 17.0346C3.1103 17.2046 3.3203 17.2846 3.5403 17.2846C3.7603 17.2846 3.9803 17.2046 4.1403 17.0346L5.9203 15.2546L6.4603 15.7946L5.8703 16.3846C5.5403 16.7146 5.5403 17.2546 5.8703 17.5846C6.0403 17.7546 6.2503 17.8346 6.4703 17.8346C6.6903 17.8346 6.9103 17.7546 7.0703 17.5846L7.6603 16.9946L8.2003 17.5346L6.4203 19.3146C6.0903 19.6446 6.0903 20.1846 6.4203 20.5146C6.5903 20.6846 6.8003 20.7646 7.0203 20.7646C7.2403 20.7646 7.4603 20.6846 7.6203 20.5146L10.1003 18.0346C10.1003 18.0346 10.1403 17.9646 10.1603 17.9346C10.2103 17.9346 10.2703 17.9346 10.3203 17.9346L11.1703 21.2346C11.3303 21.8646 11.8003 22.3446 12.4303 22.5146C12.5903 22.5546 12.7503 22.5746 12.9103 22.5746C13.3703 22.5746 13.8203 22.3946 14.1603 22.0546L16.6103 19.6046C17.0103 19.2046 17.2003 18.6346 17.1103 18.0846L16.7503 15.6746C20.5603 13.1546 23.0803 8.83459 23.0803 3.93459V3.37459C23.0803 1.76459 21.7703 0.45459 20.1603 0.45459L20.1903 0.45459ZM20.1903 2.15459C20.8603 2.15459 21.4103 2.70459 21.4103 3.37459V3.93459C21.4103 5.10459 21.2403 6.24459 20.9303 7.32459L16.2503 2.63459C17.3303 2.32459 18.4603 2.15459 19.6403 2.15459H20.2003H20.1903ZM2.6603 10.6346L2.4503 11.4546L2.6203 10.4846L5.1403 8.01459L7.0303 8.29459C6.5303 9.29459 6.1503 10.3546 5.9003 11.4646L2.6503 10.6346H2.6603ZM15.4303 18.3946L12.9803 20.8446L12.8403 20.8046L12.0303 17.6746C13.1403 17.4346 14.1903 17.0546 15.1803 16.5746L15.4303 18.4046V18.3946ZM9.3803 16.2946L7.2603 14.1846C7.3903 9.34459 10.3303 5.19459 14.5003 3.28459L20.2803 9.06459C18.3703 13.2346 14.2203 16.1646 9.3803 16.2946Z" />
```

Comment:
Same here!
This svg file is converted to tsx file by running `pnpm run icons:create`
In its tsx file you can see that it is clipPath